app/Gists/GistClient.php

```diff
@@ -2,15 +2,16 @@
 
 namespace App\Gists;
 
-use App\CachesGitHubResponses;
-use App\Exceptions\GistNotFoundException;
 use Exception;
```

Comment:
Hey, our convention at Tighten is to alphabetize import blocks, not sort by length.

I'll add our linter to Gistlog, but for now, can you fix that one?

Reply:
Oops- you know I think my vscode is automatically doing that- I'll switch it back